var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -62,11 +72,15 @@ class PyJaxlib(PythonPackage, CudaPackage):
     depends_on("bazel@4.1.0", when="@0.1.70:0.1.74", type="build")
 
     # README.md
-    depends_on("cuda@11.4:", when="@0.4:+cuda")
+    depends_on("cuda@11.8:", when="@0.4.11:+cuda")
+    depends_on("cuda@11.4:", when="@0.4.0:0.4.7+cuda")
```

Comment:
Where do these numbers come from? Want to document how to update this package in the future.

Reply:
So this is just based on:
```python
"nvidia-cuda-runtime-cu11>=11.8",                                                
"nvidia-cudnn-cu11>=8.8",
```
? Works for me

Reply:
Yes, exactly, based on that (sorry, forgot to answer)